components/src/Table/TableFoot.js

```diff
@@ -15,19 +15,37 @@ const StyledFooterRow = styled.tr({
 const renderRows = (rows, columns, keyField, loading) =>
   rows.map(row => <TableFooterRow row={row} columns={columns} key={row[keyField]} loading={loading} />);
 
-const TableFooterRow = ({ row, columns, loading }) => (
-  <StyledFooterRow>
-    {columns.map((column, index) =>
-      index === 0 ? (
-        <StyledTh key={column.dataKey} scope="row">
-          {row[column.dataKey]}
-        </StyledTh>
-      ) : (
-        !loading && <TableCell key={column.dataKey} row={row} column={column} cellData={row[column.dataKey]} />
-      )
-    )}
-  </StyledFooterRow>
-);
+const TableFooterRow = ({ row, columns, loading }) => {
+  const columnsWithoutControls = columns.filter(
+    column => column.dataKey !== "selected" && column.dataKey !== "expanded"
+  );
+  const numberOfControlColumns = columns.length - columnsWithoutControls.length;
+  // const renderEmptyCells = () =>
+  //   Array(numberOfControlColumns)
+  //     .fill()
+  //     .map(() => <td />);
+  return (
+    <StyledFooterRow>
+      {/* {renderEmptyCells()} */}
+      {columnsWithoutControls.map((column, index) =>
+        index === 0 ? (
+          <StyledTh key={column.dataKey} scope="row" colSpan={numberOfControlColumns + 1}>
```

Comment:
The commented code here is for adding in EmptyCells for the control columns.
I commented it out because to me it looked sort of strange to have the headers so far into the table.. So I added another option here using colSpan that moved the Headers to the very left.
Here are some screenshots:
COLSPAN: 
<img width="1349" alt="Screen Shot 2019-11-25 at 4 56 04 PM" src="https://user-images.githubusercontent.com/8175052/69582517-c908b400-0fa6-11ea-8623-641f66386596.png">
EMPTY TDs: 
<img width="770" alt="Screen Shot 2019-11-25 at 4 58 05 PM" src="https://user-images.githubusercontent.com/8175052/69582524-cb6b0e00-0fa6-11ea-9a13-b91d56688ac4.png">

Let me know what you think.

Reply:
I think I prefer the empty TDs because it puts the label closer to its associated content. @nikola-nulogy thoughts?

Reply:
I think I slightly prefer colspan. I like how it looks different from the data portion of the table reducing the chance of confusing it for the data. Empty td does put it closer to the content but that works only if the following column has a total value which may or may not be the case.

Reply:
Ok let's go with colspan for now then.